parts/k8s/addons/cloud-node-manager.yaml

```diff
@@ -93,7 +93,7 @@ spec:
           limits:
             cpu: 2000m
             memory: 512Mi
-{{- if HasWindows}}
+{{- if and HasWindows (IsKubernetesVersionGe "1.18.0")}}
```

Comment:
cc @chewong to sanity check that this is the only version-specific aspect of this spec

Reply:
That's right.